crates/kernel/src/vm/vm_test.rs

```diff
@@ -623,6 +624,7 @@ mod tests {
         v_list(&[v_int(4), v_int(10)]); "for list loop")]
     #[test_case(r#"if (E_INVARG == (vi = `verb_info(#-1, "blerg") ! ANY')) return 666; endif return 333;"#, 
         v_int(666); "verb_info invalid object error")]
+    #[test_case("return -9223372036854775808;", v_int(i64::MIN); "minint")]
```

Comment:
Fun fact: this whole test suite segfaults on failures. Failed to figure out why.